src/ui/welcome/index.html

```diff
@@ -214,10 +214,10 @@ <h4>About</h4>
 		<div class="intro">Gauge is a fast light-weight cross-platform test automation tool.</div>
 	</section>
 	<section class={{showGaugeInstall}}>
-		<span class="error"><i class="fas fa-exclamation-circle"></i>Gauge executable not found! </span><a href="#" class="installError"><span class="link">Read more.</span></a>
+		<span class="error"><i class="fas fa-exclamation-circle"></i>Gauge executable not found! </span>
 		<ul>
 			<li>
-				<div class="instructions hidden">
+				<div class="instructions">
```

Comment:
Has the behaviour changed to show the instructions section, instead of hiding it and show on click of a link?

Reply:
Yes, @zabil agreed to this change, to reduce interactions on the welcome page.